avm/res/compute/virtual-machine/tests/unit/az.tests.ps1

```diff
@@ -0,0 +1,15 @@
+. (Join-Path $repoRootPath 'avm' 'utilities' 'pipelines' 'sharedScripts' 'helper' 'Get-IsParameterRequired.ps1')
+
+BeforeAll {
+  $moduleJsonContentHashtable = Get-Content -Path (Join-Path 'avm' 'res' 'compute' 'virtual-machine' 'main.json') | ConvertFrom-Json -AsHashtable
+}
```

Comment:
$pesterConfiguration = @{
    Run    = @{
        Container = New-PesterContainer -Path $testFiles -Data @{
            ModuleFolderPaths = $moduleFolderPaths
            RepoRootPath      = $env:GITHUB_WORKSPACE
        }
        PassThru  = $true
    }
    Output = @{
        Verbosity = 'Detailed'
    }
}

Reply:
Note, the input for the VM module would look like this:

```Powershell
$ModuleFolderPaths  = @(
  "/home/runner/work/bicep-registry-modules/bicep-registry-modules/avm/res/compute/virtual-machine", 
  "/home/runner/work/bicep-registry-modules/bicep-registry-modules/avm/res/compute/virtual-machine/extension"
)

as seen [here](https://github.com/jtracey93/bicep-registry-modules/actions/runs/8346534040/job/22844079669#step:4:116).